src/MetaMorpho.sol

```diff
@@ -472,11 +472,15 @@ contract MetaMorpho is ERC4626, ERC20Permit, Ownable2Step, Multicall, IMetaMorph
     }
 
     /// @inheritdoc IERC4626
+    /// @dev Warning: may be lower than the actual amount of assets that can be withdrawn by `owner` due to conversion
```

Comment:
```suggestion
    /// @dev Warning: May be lower than the actual amount of assets that can be withdrawn by `owner` due to conversion
```

Reply:
this indeed corresponds to how we wrote warnings in morpho-blue